zeppelin-server/src/main/java/org/apache/zeppelin/realm/jwt/KnoxJwtRealm.java

```diff
@@ -83,14 +78,6 @@
   @Override
   protected void onInit() {
     super.onInit();
-    if (principalMapping != null && !principalMapping.isEmpty()
-        || groupPrincipalMapping != null && !groupPrincipalMapping.isEmpty()) {
-      try {
-        mapper.loadMappingTable(principalMapping, groupPrincipalMapping);
```

Comment:
`mapper` is never used anywhere else, so I removed everything related to it. Seems like something left over from copying some code from Knox.